### PR TITLE
Implemented styles for component

### PR DESCRIPTION
**Pull request by valadas**
Implemented styles for component

### CODE REVIEW

File: src/components/nvq-loan-calculator/nvq-loan-calculator.tsx [hunk 1]
```diff
@@ -174,123 +174,105 @@ export class NvqLoanCalculator {
...
           </div>
-          <p></p>
         </div>
-      </div>
-      <div class="row">
-        <div class="col">
-          <p><small><sup>*</sup> {this.interestRateFootnote}</small></p>
+        <div class="result">
+          <h4 class="text-center">{this.monthlyPaymentLabel}<sup>‡</sup></h4>
+          <span class="output">{this.calculatePayment()}$</span>
```
Comment (david-poindexter) [inline src/components/nvq-loan-calculator/nvq-loan-calculator.tsx:270]:
```suggestion
          <span class="output">${this.calculatePayment()}</span>
```

File: src/components/nvq-loan-calculator/nvq-loan-calculator.tsx [hunk 2]
```diff
@@ -174,123 +174,105 @@ export class NvqLoanCalculator {
...
-                    value={this.amortizationYears}
-                    onKeyDown={e => this.onlyAllowNumbers(e)}
-                    onInput={e => this.amortizationYears = this.getValidatedNumber(e.target as HTMLInputElement, 0)}
-                    onBlur={e => this.displayErrorIfAny(e.target as HTMLInputElement)}
-                  />
-                  <span class="error"></span>
-                </div>
-              </div>
-            </div>
+            </ul>
```
Comment (david-poindexter) [inline src/components/nvq-loan-calculator/nvq-loan-calculator.tsx:198]: Perhaps this will be in a separate PR, but these values ultimately shouldn't be hard coded, as they change often in practice and will need to be easily managed by the consumer.

**Review by david-poindexter — CHANGES_REQUESTED** [review]
Looks great!  Just a couple small things to address.

File: src/components/nvq-loan-calculator/nvq-loan-calculator.tsx (hunk 1 above)
Comment (valadas) [reply src/components/nvq-loan-calculator/nvq-loan-calculator.tsx:270]: Good catch

File: src/components/nvq-loan-calculator/nvq-loan-calculator.tsx (hunk 2 above)
Comment (valadas) [reply src/components/nvq-loan-calculator/nvq-loan-calculator.tsx:198]: Sure, I had 2 ideas:
1. Make an interface for one example `ILoanExample` and for the component to accept an array of them.
2. Make that a slot so consumers can just push whatever content they want.

Since that part has no logic at all I was thinking a slot would be the simplest, thoughts ?

Comment (david-poindexter) [reply src/components/nvq-loan-calculator/nvq-loan-calculator.tsx:198]: The more I think about it the more I think your slot idea is best.  Let's go with that.
